generator.yaml

```diff
@@ -485,6 +485,43 @@ resources:
         from:
           operation: DescribeMonitoringSchedule
           path: LastMonitoringExecutionSummary
+  FeatureGroup:
+    exceptions:
+      errors:
+          404:
+            code: ResourceNotFound
+      terminal_codes:
+        - ResourceInUse
+        - ResourceLimitExceeded
+        - ResourceNotFound
+        - InvalidAction
+        - InvalidClientTokenId
+        - InvalidParameterCombination
+        - InvalidParameterValue
+        - InvalidQueryParameter
+        - MalformedQueryString
+        - MissingAction
+        - MissingParameter
+        - OptInRequired
+    hooks:
+      delta_pre_compare:
+        code: customSetDefaults(a, b)
+    fields:
+      FailureReason:
+        is_read_only: true
+        print:
+          name: FAILURE-REASON
```

Comment:
did we decide to change this column name across resources ?

Reply:
This update happened in [PR #44](https://github.com/aws-controllers-k8s/sagemaker-controller/pull/44/files).